app/controllers/organizations/staff_controller.rb

```diff
@@ -1,9 +1,31 @@
 class Organizations::StaffController < Organizations::BaseController
-  layout "dashboard"
-
   before_action :require_organization_admin
+  layout "dashboard"
 
   def index
     @staff_accounts = StaffAccount.all
   end
+
+  def new
+    @staff = StaffAccount.new(user: User.new)
+  end
+
+  def create
+    @user = User.new(staff_params[:user].merge(password: SecureRandom.hex(8)))
+    @staff = StaffAccount.new(user: @user, verified: true)
```

Comment:
I think we can simplify this using nested form approach with accept_nested_attributes - https://www.ombulabs.com/blog/learning/rails/nested-forms.html